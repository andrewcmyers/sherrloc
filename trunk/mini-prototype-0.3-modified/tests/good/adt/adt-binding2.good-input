type t : * -> * =  K : forall a. a -> t a

let z =  let y = K 0 in match y with K x => x end 
