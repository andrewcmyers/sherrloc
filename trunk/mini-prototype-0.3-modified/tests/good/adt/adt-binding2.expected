val K: forall a. a -> t a
val z: int

